api/filter/BUILD

```diff
@@ -15,6 +15,9 @@ api_proto_library(
 api_proto_library(
     name = "mongo_proxy",
     srcs = ["mongo_proxy.proto"],
+    deps = [
```

Comment:
`deps = [":http_fault"],` for singleton. Please run `buildifier` over this file.